README.md

```diff
@@ -40,6 +40,10 @@ HLS stream, code against the regular HTML5 video APIs, and create a
 fast, high-quality video experience across all the big web device
 categories.
 
+Check out the [full documentation](docs/) for details on how HLS works
```

Comment:
That's a good point. There's a bunch in the mp2t-to-mp4 branch that I'd like to move into master someday...